cypress/integration/collections_spec.js

```diff
@@ -193,13 +190,15 @@ describe('Dashboard Collections Page', () => {
       cy.url().should('include', 'collections');
       cy.contains('.heading--xlarge', 'Collections');
       // verify the collection is now gone
+      cy.get('[data-value="http_testcollection___001"] > .table__main-asset > a').should('exist');
```

Comment:
This is still giving me problems, and I will continue to look at it, but it's a race condition somewhere that I need to find.  the tests pass most of the time (and always when I do it by hand).

Reply:
This problem is resolved by both forcing a refresh and increasing the timeout.